.github/workflows/release-from-main.yml

```
@@ -0,0 +1,95 @@
+name: Merge to master
+
+on:
+  push:
+    branches:
+      - master
```

```suggestion
  push:
    branches:
      - master
    paths:
      - VERSION
```

Because this workflow differs to the way Android works which builds on every commit to main we can add the limit to only run the workflow if the VERSION file changed

---

Thanks, done